cumulus_library/studies/discovery/code_detection.py

```diff
@@ -1,39 +1,40 @@
 """ Module for generating encounter codeableConcept table"""
 
-from cumulus_library import base_table_builder, helper
-from cumulus_library.template_sql import templates, utils
-
+from cumulus_library import base_table_builder, base_utils
 from cumulus_library.studies.discovery import code_definitions
+from cumulus_library.template_sql import base_templates, sql_utils
 
 
-class CodeDetectionBuilder(BaseTableBuilder):
+class CodeDetectionBuilder(base_table_builder.BaseTableBuilder):
     display_text = "Selecting unique code systems..."
 
     def _check_codes_in_fields(self, code_sources: list[dict], schema, cursor) -> dict:
         """checks if Coding/CodeableConcept fields are present and populated"""
 
-        with helper.get_progress_bar() as progress:
+        with base_utils.get_progress_bar() as progress:
             task = progress.add_task(
                 "Discovering available coding systems...",
                 total=len(code_sources),
             )
             for code_source in code_sources:
                 if code_source["is_array"]:
-                    code_source["has_data"] = utils.is_codeable_concept_array_populated(
+                    code_source[
+                        "has_data"
+                    ] = sql_utils.is_codeable_concept_array_populated(
```

Comment:
This construct doesn't feel very readable to me.

Maybe a temporary variable would be better? Like `has_data = sql_utils...(` and then `code_source["has_data"] = has_data`

But small nit - it's fine as is.

Reply:
related to my 'mccabe complexity is a headache' comment, i just went ahead and moved this logic to a function so the line length wasn't an issue.